### ISSUE TITLE
Can we remove the `.Net Core Tools` package source?

### ISSUE THREAD

**Issue by luizhlelis** [open-telemetry/opentelemetry-dotnet#4403]
# Question

Hello everyone.

I'm working on #4298, and the following error was thrown after central package management because `opentelemetry.net` is using more than one package source:

```sh
error NU1507: Warning As Error: There are 2 package sources defined in your configuration.
When using central package management, please map your package sources with package
source mapping (https://aka.ms/nuget-package-source-mapping) or specify a single package source.
The following sources are defined: NuGet, .Net Core Tools 
```

@TimothyMothra said that probably we're not using the `.Net Core Tools` package source anymore and suggested creating a new issue. Can we remove the `.Net Core Tools` package source?

[comment reference](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4298#discussion_r1169049440)
[package source reference](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/NuGet.config#L9)

**Comment by TimothyMothra** [comment]
> @TimothyMothra said that probably we're not using the .Net Core Tools package source anymore

To clarify, we are currently using this package source but we probably don't need this anymore.
This was added in 2021 to take a beta version of the ApiCompat tool.
However, this tool is now available on Nuget.org.

**Comment by utpilla** [comment]
@TimothyMothra Need to confirm if the new `DotNet.ApiCompat.Tool` on NuGet is actually the equivalent of the package `Microsoft.DotNet.ApiCompat` from .NET Core Tools feed.

Also, since the [recommendation from the .NET team](https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.ApiCompat) is to use PackageValidation tool, it wouldn't really help much to spend time exploring `DotNet.ApiCompat.Tool` from NuGet. We should explore the possibility of replacing the existing `Microsoft.DotNet.ApiCompat` check with Package Validation tool.

**Comment by luizhlelis** [comment]
Thank you, guys.

To unblock [this PR](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4298), I'll not remove the `.Net Core Tools package` source, on the other hand, I'm adding the following package pattern to be restored by that source:

```xml
<package pattern="*.ApiCompat" />
```
